token/cli/src/main.rs

```diff
@@ -3050,7 +3084,7 @@ fn process_command(
                 bulk_signers,
             )
         }
-        ("sync-native", arg_matches) => {
+        (CommandName::SyncNative, arg_matches) => {
             let address = config.associated_token_address_for_token_or_override(
                 arg_matches,
                 "address",
```

Comment:
I can't comment lower, but can you remove the final case?
```
_ => unreachable!(),
```
With this change, we can be explicit!